hack/make.ps1

```diff
@@ -356,7 +350,7 @@ Try {
     if ($CommitSuffix -ne "") { $gitCommit += "-"+$CommitSuffix -Replace ' ', '' }
 
     # Get the version of docker (eg 17.04.0-dev)
-    $dockerVersion=Get-DockerVersion
+    $dockerVersion="0.0.0-dev"
 
```

Comment:
why not "dev"?

Reply:
Mostly to cause the least disruption (there may / will still be code parsing the version)

Reply:
The 0.0.0 is necessary here because of `go-autogen`. In windows it parses the version into a comma separated string, which it injects into some "windows service" config files. Those files were reporting a syntax error when I used just `dev`, so I had to add `0.0.0`.

Reply:
could you add that comment to make.ps1?